packages/core/strapi/lib/types/core/config/index.d.ts

```diff
@@ -1,4 +1,5 @@
 export type { Server } from './server';
 export type { Admin } from './admin';
 export type { Api } from './api';
+export type { PluginConfigs } from './plugin';
```

Comment:
Is there a reason why it's not just Plugin or Plugins? (actually, same question for database)

Reply:
As long as we stay consistent in all the different type files, I'm ok either way. Config suffix is a bit redundant since we're already in a config, but I don't mind it if it's clearer.